speeduino/idle.cpp

```diff
@@ -485,6 +484,11 @@ void idleControl(void)
       {
         //Currently cranking. Use the cranking table
         currentStatus.idleLoad = table2D_getValue(&iacCrankDutyTable, currentStatus.coolant + CALIBRATION_TEMPERATURE_OFFSET); //All temps are offset by 40 degrees
+        //Add Idle Up amount if active
```

Comment:
This block of code is repeated many times. Could you hoist it into a `static inline ` function? E.g. `getIdelUpLoad(void)`